services/MerchStoreService.ts

```diff
@@ -540,4 +540,11 @@ export default class MerchStoreService {
       await orderPickupEventRepository.deletePickupEvent(pickupEvent);
     });
   }
+
+  public async getCartItems(items: string[]): Promise<Map<string, MerchandiseItemOptionModel>> {
+    return this.transactions.readWrite(async (txn) => {
+      const merchItemRepository = Repositories.merchStoreItemOption(txn);
+      return merchItemRepository.batchFindByUuid(items);
```

Comment:
1. How should we handle items in archived collections or hidden items?
2. How should we handle missing items? Throw or only return found items?

Reply:
I don't think we need to be returning items that members don't have permissions for, like Sumeet said. If the frontend wants to let the user know that some of their cart items have been archived/hidden, then frontend can diff the uuids sent and the uuids received and display something like "Some of the items in your cart have recently been archived."

Reply:
> If the frontend wants to let the user know that some of their cart items have been archived/hidden, then frontend can diff the uuids sent and the uuids received and display something like "Some of the items in your cart have recently been archived."

We can include this diff in `GetCartResponse`.